code/enumerable/inject-symbol-vs-block.rb

```diff
@@ -0,0 +1,24 @@
+require "rubygems"
+require "benchmark/ips"
+
+ARRAY = (1..1000).to_a
+
+def fastest
```

Comment:
Yeah, sorry. I see now that that could be read as an attack. I just meant the convention in general should change, IMHO.